### PR TITLE
Add Windows blog post!

### PR DESCRIPTION
**Pull request by toddboom**

### CODE REVIEW

File: content/blog/2015/10/09/windows_installer_times_series_database.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+* Chronograf Dashboard - Opens the local Chronograf Dashboard.
+* Documentation - Links to the online documentation for Chronograf.
+
+![](/img/blog/windows/image09.png)
+
+Launching the Chronograf Server will result in the following screen:
+
+![](/img/blog/windows/image05.png)
+
+Launching the Chronograf Dashboard will open a browser.  If you have not registered with Influxdata yet, you will be promoted to sign up before you can use Chronograf.
```
Comment (otoolep) [inline content/blog/2015/10/09/windows_installer_times_series_database.md]: Influxdata -> influxdata.com? And use a complete URL?

File: content/blog/2015/10/09/windows_installer_times_series_database.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+![](/img/blog/windows/image16.png)
+
+After completing the singup, you will be redirected to your local dashboard.  You will first be prompted to add your local server.  You can accept all the default options.
```
Comment (otoolep) [inline content/blog/2015/10/09/windows_installer_times_series_database.md]: signup -> signup

File: content/blog/2015/10/09/windows_installer_times_series_database.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
++++
+title = "Announcing Windows Installation Packages for InfluxDB"
+author = "Cory LaNou"
+date = "2015-10-09"
+publishdate = "2015-10-09"
++++
+
+Today we are releasing our initial Windows install packages for [InfluxDB](https://influxdb.com/), [Telegraf](https://github.com/influxdb/telegraf),  and [Chronograf](https://influxdb.com/chronograf/).  These packages represent a minimal installation designed get developers up and running quickly with local instances. Future releases will likely include the ability to:
```
Comment (otoolep) [inline content/blog/2015/10/09/windows_installer_times_series_database.md]: designed -> designed *to*

File: content/blog/2015/10/09/windows_installer_times_series_database.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+
+![](/img/blog/windows/image13.png)
+
+![](/img/blog/windows/image07.png)
+
+After the installation you will see a new folder in your Startup Menu called “InfluxDB”.  The following shortcuts are installed for your convenience:
+
+* InfluxDB Server - Launches the InfluxDB database service.
+* InfluxDB CLI - Launches the command line interface used to query and communicate with the InfluxDB server.
+* Admin Dashboard - Opens your local dashboard used to communicate with your InfluxDB database.
```
Comment (otoolep) [inline content/blog/2015/10/09/windows_installer_times_series_database.md]: Very minor: "your local" is a little vague here. Perhaps just "the dashboard"?

File: content/blog/2015/10/09/windows_installer_times_series_database.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
...
+
+Launching the InfluxDB server you should see the following screen:
+ 
+![](/img/blog/windows/image14.png)
+
+Launching the InfluxDB CLI should result in the following screen:
+
+![](/img/blog/windows/image17.png)
+
+The InfluxDB CLI will automatically connect to your local InfluxDB instance if it is running.  The above example shows the command output for `show databases`.
```
Comment (otoolep) [inline content/blog/2015/10/09/windows_installer_times_series_database.md]: Should we capitalize `SHOW DATABASES`?

File: content/blog/2015/10/09/windows_installer_times_series_database.md [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+You can also launch the Admin Dashboard for a GUI experience using the Admin Dashboard shortcut.  This will open your browser to the local Admin Dashboard and automatically connect to your local InfluxDB instance if it is running.
```
Comment (otoolep) [inline content/blog/2015/10/09/windows_installer_times_series_database.md]: your browser *pointing at* the local Admin dashboard?

**Comment by corylanou** [comment]
@beckettsean @rkuchan Looks like this got merged prematurely?  /cc @jginfluxdata

**Comment by beckettsean** [comment]
@corylanou maybe. What's the best way through? Revert the merge?

**Comment by corylanou** [comment]
@beckettsean I would just issue another small pr

**Comment by gunnaraasen** [comment]
I'll open another PR to address @otoolep's comments.

**Pull request merged**